easybuild/easyconfigs/j/jupyter-resource-usage/jupyter-resource-usage-1.0.0-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonBundle'
+
+name = 'jupyter-resource-usage'
+version = "1.0.0"
+
+homepage = 'https://github.com/jupyter-server/jupyter-resource-usage'
+description = "Jupyter Notebook Extension for monitoring your own Resource Usage (memory and/or CPU)"
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+builddependencies = [
+    ('binutils', '2.40'),
+]
+dependencies = [
+    ('Python', '3.11.3'),
+    ('IPython', '8.14.0'),
+    ('jupyter-server', '2.7.2'),
+]
+
+sanity_pip_check = True
+use_pip = True
+
+exts_list = [
+    ('jupyter_resource_usage', version, {
+        'checksums': ['74b35b5040c6034e06062f72f8bc5b4a473549e009151be488dd3f61866b854b'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/jupyter_resource_usage'],
+}
+
+# Add the notebook extension to the search path for jupyter notebooks
+modextrapaths = {'JUPYTER_PATH': 'share/jupyter/'}
```

Comment:
add `share/jupyter` to `dirs` in `sanity_check_paths`?

Reply:
Done